raddb/mods-available/eap

```diff
@@ -67,111 +80,135 @@ eap {
+	#  or the `NT-Password` attributes. 'System' authentication is impossible
 	#  with LEAP.
+	#  ====
 	#
 #	leap {
 #	}
 
-	## Generic Token Card.
 	#
-	#  Currently, this is only permitted inside of EAP-TTLS, or EAP-PEAP.
-	#  The module "challenges" the user with text, and the response from the
-	#  user is taken to be the User-Password.
+	#  ### Generic Token Card
+	#
+	#  gtc { ... }::
 	#
-	#  Proxying the tunneled EAP-GTC session is a bad idea, the users
+	#  Currently, this is only permitted inside of `EAP-TTLS`, or `EAP-PEAP`.
+	#  The module `challenges` the user with text, and the response from the
+	#  user is taken to be the `User-Password`.
+	#
+	#  WARNING: Proxying the tunneled `EAP-GTC` session is a bad idea, the users
 	#  password will go over the wire in plain-text, for anyone to see.
 	#
 	gtc {
-		#  The default challenge, which many clients ignore..
+		#
+		#  challenge:: The default challenge, which many clients ignore.
+		#
 #		challenge = "Password: "
 
+		#
+		#  auth_type::
 		#
 		#  The plain-text response which comes back is put into a
-		#  User-Password attribute, and passed to another module for
-		#  authentication.  This allows the EAP-GTC response to be
+		#  `User-Password` attribute, and passed to another module for
+		#  authentication.  This allows the `EAP-GTC` response to be
 		#  checked against plain-text, or crypt'd passwords.
 		#
 		#  If you say "Local" instead of "PAP", then the module will
-		#  look for a User-Password configured for the request, and do
+		#  look for a `User-Password` configured for the request, and do
 		#  the authentication itself.
 		#
 		auth_type = PAP
 	}
 
-	## Common TLS configuration for TLS-based EAP types
 	#
-	#  See raddb/certs/README for additional comments on certificates.
+	#  ## Common TLS configuration for TLS-based EAP types
 	#
-	#  If OpenSSL was not found at the time the server was built, the "tls",
-	#  "ttls", and "peap" sections will be ignored.
+	#  See `raddb/certs/README` for additional comments on certificates.
+	#
+	#  If OpenSSL was not found at the time the server was built, the `tls`,
+	#  `ttls`, and `peap` sections will be ignored.
 	#
 	#  If you do not currently have certificates signed by a trusted CA you
 	#  may use the 'snakeoil' certificates. Included with the server in
-	#  raddb/certs.
+	#  `raddb/certs`.
 	#
 	#  If these certificates have not been auto-generated:
 	#
 	#    cd raddb/certs
 	#    make
 	#
-	#  These test certificates SHOULD NOT be used in a normal deployment.
+	#  These test certificates *SHOULD NOT* be used in a normal deployment.
 	#  They are created only to make it easier to install the server, and to
-	#  perform some simple tests with EAP-TLS, TTLS, or PEAP.
+	#  perform some simple tests with `EAP-TLS`, `TTLS`, or `PEAP`.
 	#
-	#  Note that you should NOT use a globally known CA here! e.g. using a
+	#  NOTE: You should NOT use a globally known CA here! e.g. using a
```

Comment:
The contents of this note should be removed or moved.  Cert verification no longer works like this.